parent/pom.xml

```diff
@@ -2854,35 +2878,38 @@
                     <artifactId>javax.annotation-api</artifactId>
                 </dependency>
                 <dependency>
-                    <groupId>javax.xml.ws</groupId>
-                    <artifactId>jaxws-api</artifactId>
-                    <version>2.3.1</version>
-                    <exclusions>
-                        <exclusion>
-                            <groupId>javax.xml.bind</groupId>
-                            <artifactId>jaxb-api</artifactId>
-                        </exclusion>
-                    </exclusions>
+                    <groupId>jakarta.xml.ws</groupId>
+                    <artifactId>jakarta.xml.ws-api</artifactId>
+                    <version>2.3.2</version>
                 </dependency>
                 <dependency>
                     <groupId>com.sun.activation</groupId>
                     <artifactId>javax.activation</artifactId>
                     <version>1.2.0</version>
                 </dependency>
                 <dependency>
-                    <groupId>org.apache.geronimo.specs</groupId>
-                    <artifactId>geronimo-ws-metadata_2.0_spec</artifactId>
-                    <version>1.1.3</version>
+                    <groupId>jakarta.jws</groupId>
+                    <artifactId>jakarta.jws-api</artifactId>
+                    <version>1.1.1</version>
+                </dependency>
+                <dependency>
+                    <groupId>jakarta.xml.soap</groupId>
+                    <artifactId>jakarta.xml.soap-api</artifactId>
+                    <version>1.4.1</version>
                 </dependency>
                 <dependency>
                     <groupId>com.sun.xml.messaging.saaj</groupId>
                     <artifactId>saaj-impl</artifactId>
-                    <version>1.4.0-b03</version>
+                    <version>1.3.23</version>
```

Comment:
The SAAJ (impl) was one of the problems we run into before, so we have now:
 - `1.5.1` in samples/pom.xml
 - `1.3.23` in parent/pom.xml

Meantime, SAAJ API is 1.4.0.1:
 - `<cxf.osgi.saaj.version>[1.4,2)</cxf.osgi.saaj.version>` in parent/pom.xml

Should we keep `1.4.0-b03` to align API and impl? It seemed to work fine.

Reply:
Good catch. For me 1.5.1 also works, but 1.4 is fine as well. @ffang what issues did you have?

Reply:
Just checked 1.5.2-b01, also works.

Reply:
@deki , sorry, can't recall exactly, but should be OSGi test related
I suggest we use saaj api 1.4 from SMX and use saaj 1.3 impl

Thanks!